src/functions/utils.php

```diff
@@ -695,16 +694,20 @@ function tribe_get_request_vars( $refresh = false ) {
 	 *                    array.
 	 */
 	function tribe_sanitize_deep( &$value ) {
+		if ( is_bool( $value ) ) {
+			$value = filter_var( $value, FILTER_VALIDATE_BOOLEAN );
```

Comment:
This is less restrictive than the is_bool call so I don’t know how useful the filter_var is right here. I think you can just return here instead.

Reply:
That is true. haha